src/steps/build.py

```diff
@@ -94,6 +94,21 @@ def has_build_script(script_name):
     return False
 
 
+def is_supported_ruby_version(version):
+    '''
+    Checks if the version defined in .ruby-version is supported
+    '''
+    default_version = os.getenv('RUBY_VERSION')
+    major_version = int(default_version.split('.')[0])
+    minor_version = int(default_version.split('.')[1])
+
+    semver = version.split('.')
```

Comment:
We removed `shlex.quote()` and are manually parsing the version here. We don't do this for the node version and use `nvm version-remote` instead because folks specify versions in all sorts of ways (ex `v12`), Is Ruby (or rvm) more strict about what it accepts as a valid version specifier? `jruby-9.3.3.0` is apparently a valid version, though we don't support it, would want to make sure we have a meaning message in this case, not a parsing error.

Reply:
Now using ruby's `Gem::Version.new` which will allow for interpreting known versions.